app/Helpers/ImageHelpers.php

```diff
@@ -680,4 +680,13 @@ public static function aic_getIconClass($key = 0)
 
         return Str::kebab($icons[$key]);
     }
+    public static function getSocialIcon($url)
+    {
+        $pattern = '/(?:https?:\/\/)?(?:www\.)?([a-zA-Z0-9.-]+)\.(com|net|org)/i';
+
+        // Use preg_match to find the domain in the URL
+        if (preg_match($pattern, $url, $matches)) {
+            return $matches[1];
+        }
```

Comment:
I think this function could be simplified with [`parse_url`](https://www.php.net/manual/en/function.parse-url.php), for example:
```php
$host = parse_url($url,  PHP_URL_HOST);
// Split host into sub domain(s), host domain, and top-level domain
$domains = explode('.', $host);
// The second-to-last domain item is the host domain
return $domains[count($domains) - 2];
```